fdbserver/CMakeLists.txt

```
@@ -277,6 +277,7 @@ else()
 endif()
 
 target_link_libraries(fdbserver PRIVATE toml11_target)
+target_link_libraries(fdbserver PRIVATE jemalloc)
```

Nit: is there a reason this is on two lines? I would prefer all in one:
```suggestion
target_link_libraries(fdbserver PRIVATE toml11_target jemalloc)
```